### ISSUE TITLE
Video removeProcessor not compatible with @twilio/video-processors release candidate

### ISSUE THREAD

**Issue by ianvonholt** [twilio/twilio-video.js#1964]
- [x] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [x] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [x] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.
 - [x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Code to reproduce the issue:**

- Install / clone `@twilio/video-processors` at `2.0.0-rc1`;
- Utilizing the video-processors example, update the `twilio-video` script to utilize the newest release:
https://github.com/twilio/twilio-video-processors.js/blob/2.0.0-rc1/examples/virtualbackground/index.html#L44

Removing a processor will then throw an error that `clearRect` is not present on object of null. 

It looks like `this._outputFrame` does not get a context at all. This can be fixed by creating a dummy canvas prior to removal:

```js
videoTrack._outputFrame = document.createElement('canvas');
videoTrack.removeProcessor(videoTrack.processor);
```

There was work done for cross browser compatibility, but it looks like some of that code was not included in further releases: 
https://github.com/twilio/twilio-video.js/tree/2.23.1-rc1 

**Expected behavior:**

Removal of the video processor regardless of `_inputFrame` or `_outputFrame` existing.

**Actual behavior:**

Video freezes when a processor, generated with the `video-processors` release candidate, is removed. 

**Software versions:**

 - [x] Browser(s): Chrome, Firefox, Safari
 - [x] Operating System: macOS
 - [x] twilio-video.js: 2.26.0
 - [x] Third-party libraries (e.g., Angular, React, etc.): `@twilio/video-processors` at `2.0.0-rc1`

**Comment by seancoleman2** [comment]
@ianvonholt thanks for surfacing this. We will be revisiting `@twilio/video-processor` `2.0.0-rc1` this quarter to work to get it in a releasable state. We plan to take a look at this issue when we start on that work. 

cc: @anna-vasilko @manjeshbhargav internal ticket: VIDEO-12409

**Comment by pingumen96** [comment]
Hello everyone, I'm sorry because, this comment will not be a contribution to the issue, I just want to thank @ianvonholt for the workaround: you saved my day.
@twilio guys, is there some news about the release date of twilio/video-processors v2? We are already using the rc because of the improvements on the model of the background video processor and we are going to strongly rely on it, thank you for your work

**Comment by ianvonholt** [comment]
@seancoleman2 Let me know if I can help out. We'll be using this in production, I wouldn't mind doing a 20% project to get this out to the masses.

**Comment by manjeshbhargav** [comment]
Hi @ianvonholt ,

We have released @twilio/video-processors@2.0.0, and it should be used in conjunction with twilio-video@2.27.0. Please let me know if it works for you. I will close this issue, but we can continue the conversation here.
